src/views/CardCreatorPage.vue

```diff
@@ -882,7 +882,11 @@ export default {
       return printString.length > 1 ? R.dropLast(1, printString) : "";
     },
     showBuyFrameModal() {
-      this.isBuyFrameModalVisible = true;
+      if (!this.address) {
+        this.notifyFail("Buy Card Frame failed", "You must be logged in with an activated account for this.");
```

Comment:
I would suggest going with something like. `"Unable to buy card"`. That would indicate, this this isn't an error, but rather a restriction.